src/main/java/org/broadinstitute/hellbender/tools/copynumber/formats/collections/ChunkedCopyNumberPosteriorCollection.java

```diff
@@ -0,0 +1,56 @@
+package org.broadinstitute.hellbender.tools.copynumber.formats.collections;
+
+import org.broadinstitute.hellbender.exceptions.UserException;
+import org.broadinstitute.hellbender.tools.copynumber.gcnv.CopyNumberPosteriorRecord;
+import org.broadinstitute.hellbender.tools.copynumber.gcnv.IntegerCopyNumberState;
+import org.broadinstitute.hellbender.tools.copynumber.gcnv.IntegerCopyNumberStateCollection;
+import org.broadinstitute.hellbender.utils.Utils;
+import org.broadinstitute.hellbender.utils.tsv.DataLine;
+
+import java.io.File;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.function.BiConsumer;
+import java.util.function.Function;
+
+/**
+ * Collection of copy number posteriors for an individual chunk containing a subset of intervals considered in the analysis
+ */
+public class ChunkedCopyNumberPosteriorCollection extends AbstractSampleRecordCollection<CopyNumberPosteriorRecord> {
+
+    public ChunkedCopyNumberPosteriorCollection(final File inputFile,
```

Comment:
To be consistent with the other collection classes, you can put the constructor at the end after the static codecs.